projects/media-viewer/src/lib/toolbar/main-toolbar/main-toolbar.component.html

```diff
@@ -5,9 +5,7 @@
     <div id="toolbarViewer">
       <mv-tb-left-pane></mv-tb-left-pane>
       <mv-tb-middle-pane></mv-tb-middle-pane>
-      <mv-tb-right-pane
-        [enableAnnotations]="enableAnnotations">
-      </mv-tb-right-pane>
+      <mv-tb-right-pane></mv-tb-right-pane>
```

Comment:
does this not require the enableAnnotations input any longer? was that used for the comment summary?

Reply:
correct... no needed any longer